### ISSUE TITLE
MessageLogged Event needs log level, not just log name

### ISSUE THREAD

**Issue by ChangePlaces** [laravel/internals#913]
- Laravel Version: 5.5

When working with logs, it's typical to want to analyse the log level. This is impossible with the logged message event that only passes the log level name (e.g. 'error') instead of a numerical value as well, so simple logic like `if ($level >= \Monolog\Logger::ERROR)` can be performed. Also, regarding monolog, `$level` refers to the numerical value, and `$level_name` refers to the actual name attributed to it.

I think a log_level member should be added to the event for this reason.

**Comment by sisve** [comment]
Renaming the existing $level to $level_name would be a breaking change.

**Comment by ChangePlaces** [comment]
Yep, that's why I'm not suggesting level is changed to level_name, but another property added called log_level which is the int value
